sotodlib/io/load_smurf.py

```diff
@@ -692,14 +700,26 @@ def add_new_observation(
                     )
 
             # Build Observation
-            obs = Observations(
-                obs_id=f"{stream_id}_{session_id}",
-                timestamp=session_id,
-                action_ctime=action_ctime,
-                action_name=action_name,
-                stream_id=stream_id,
-                start=start,
-            )
+            if calibration:
```

Comment:
could you change this to something like: 
```
if calibration:
    obs_id = f"cal_{stream_id}_{session_id}"
else:
    obs_id=f"{stream_id}_{session_id}"
    
obs = Observations(
    obs_id=obs_id,
    timestamp=session_id,
    action_ctime=action_ctime,
     action_name=action_name,
    stream_id=stream_id,
    start=start,
    calibration=calibration,
)
```

This makes it much easier to keep track of how different variables are set. And easier to change with time if we need to edit how observations are made in the future.